git-and-github/code-reviews.md

```diff
@@ -52,10 +53,12 @@
 
         * Don’t forget to compliment the submitter on good stuff, too!
 
-2. Mark your review as one of the following:
-    `Request Changes`: If there are bugs in the code or something unacceptable being introduced to the code base (swearing in comments, console references and debuggers, etc.), you should block the PR and leave detailed comments as to why. If you block a PR and the author resubmits changes, get to them as quickly as you can and approve the PR once it's addressed
-    `Approve`: If the code looks good and you have no questions/concerns, or if the author has submitted fixes to a blocking PR, mark it as approved and Squash & Merge or just Merge the PR and delete the compare branch.
-    `Comment`: If you have comments/concerns but you are not blocking the author (no bugs) then mark the review as comment and leave your comments. Let the author know that they can merge if they don't want to address your concerns.
+2. Mark your review as one of the following:  
+    * `Request Changes`: If there are bugs in the code or something unacceptable being introduced to the code base (swearing in comments, console references and debuggers, etc.), you should block the PR and leave detailed comments as to why. If you block a PR and the author resubmits changes, get to them as quickly as you can and approve the PR once it's addressed.  
+    * `Approve`: If the code looks good and you have no questions/concerns, or if the author has submitted fixes to a blocking PR, mark it as approved.  
+    * `Comment`: If you have comments/concerns but you are not blocking the author (no bugs) then mark the review as comment and leave your comments. Let the author know that they can merge if they don't want to address your concerns.
+
+3. If you are a maintainer for the project you can Squash & Merge or just Merge the PR and delete the compare branch once it is approved. If you are not a maintainer for the project, do NOT merge the PR. Leave it for the author or maintainers.
```

Comment:
I would bold the and maybe capitalize the `do NOT merge the PR` part.

Reply:
Yeah bold and capitalize may be good, that way someone can't say they didn't see it

Reply:
happy to oblige